Readme.md

```diff
@@ -84,17 +84,19 @@ https://codepen.io/robstinson/pen/BaKOZry
 
 ---
 
-# Things to add
-
-- Create a `.env` file in the config folder and add the following as `key = value.`
-  - PORT = 2121 (Do not change)
-  - DB_STRING = `your database URI`
-  - CLOUD_NAME = `your Cloudinary cloud name`
-  - API_KEY = `your Cloudinary API key`
-  - API_SECRET = `your Cloudinary API secret`
-  - DISCORD_CLIENT_ID = `discord client ID`
-  - DISCORD_CLIENT_SECRET = `discord client secret`
-
+# Things edit 
+
+
+- Rename the .env.example to just .env and include the following
```

Comment:
What was the intention behind this change? The readme looks like this now. 
<img width="1004" alt="Screen Shot 2022-11-12 at 3 24 30 PM" src="https://user-images.githubusercontent.com/75268174/201495124-53676e76-4c43-4223-bc4b-16d183006008.png">

Reply:
I put it in my one of the atomic commits but I'll just say it again to kinda solidify it. Don't leave spaces between the decleration of the environment variable, the equal sign between it and the its value and you might have guessed it but no space in the thing after the equals sign.